pkg/controller/component/component_controller.go

```diff
@@ -127,27 +130,15 @@ func (r *ReconcileComponent) Reconcile(request reconcile.Request) (reconcile.Res
 			log.Error(err, "** Setting owner reference fails **")
 			return reconcile.Result{}, err
 		}
-
-		// Create an empty image name "nodejs-runtime"
-		// todo(corinne): check if we can reuse openshift's image stream
-		newImageForRuntime := newImageStreamFromDocker(instance.Namespace, instance.Name, instance.Spec.BuildType)
-		err = r.client.Create(context.TODO(), newImageForRuntime)
+		// Create a build image named either "myapp-runtime" or reuse openshift "nodejs" runtime
+		ir, err := r.getImageRuntime(instance)
```

Comment:
Could you please call it `getBuilderImage(..)` ? The reason I try to specify the difference between `builder` and `runtime` is that some languages do not have the same image requirements at build and at runtime.

Example, Golang apps need a golang builder image but doesn't need a go runtime.

Reply:
fixed in 6a1332a

Reply:
6a1332a